tests-beta/android/MASVS-NETWORK/MASTG-TEST-0x22-3.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Missing Certificate Pinning in Network Traffic 
+platform: network
+id: MASTG-TEST-0x22-3
```

Comment:
Could you please update each test and give it a unique ID like here: https://github.com/OWASP/owasp-mastg/tree/master/tests-beta/android/MASVS-NETWORK. This means also changing the filename accordingly.